features/measurements/impl/src/test/java/org/opennms/netmgt/measurements/filters/impl/LoadDataTrendLineTest.java

```diff
@@ -0,0 +1,156 @@
+/*******************************************************************************
```

Comment:
This file still doesn't comply with our checkstyle, branch will fail CI pipeline.